ddev-instances/core-12/composer.json

```diff
@@ -1,92 +1,93 @@
+        "typo3/minimal": "^12"
     },
-    "sort-packages": true
-  },
-  "extra": {
-    "typo3/cms": {
-      "web-dir": "public"
+    "config": {
+        "allow-plugins": {
+            "typo3/class-alias-loader": true,
+            "typo3/cms-composer-installers": true,
+            "php-http/discovery": true,
+            "sbuerk/fixture-packages": true
+        },
+        "sort-packages": true
     },
-    "branch-alias": {
-      "dev-main": "2.0.x-dev"
-    }
-  },
-  "require-dev": {
-    "bnf/phpstan-psr-container": "^1.0.1",
-    "friendsofphp/php-cs-fixer": "^3.57.1",
-    "friendsoftypo3/phpstan-typo3": "^0.9.0",
-    "georgringer/numbered-pagination": "^2.1",
-    "phpstan/phpdoc-parser": "^1.29.0",
-    "phpstan/phpstan": "^1.12.21",
-    "phpstan/phpstan-phpunit": "^1.4.0",
-    "phpunit/phpunit": "^10.5.45",
-    "sbuerk/fixture-packages": ">=0.1.1 < 2.0.0",
-    "typo3/tailor": "^1.6",
-    "typo3/testing-framework": "^8.2.7"
-  },
-  "repositories": {
-    "packages-dev": {
-      "type": "path",
-      "url": "../packages-dev/*",
-      "options": {
-        "versions": {
-          "fgtclb/academics-monorepo-shared": "2.0.2-dev"
+    "extra": {
+        "typo3/cms": {
+            "web-dir": "public"
+        },
+        "branch-alias": {
+            "dev-main": "2.0.x-dev"
+        }
+    },
+    "require-dev": {
+        "bnf/phpstan-psr-container": "^1.0.1",
+        "friendsofphp/php-cs-fixer": "^3.57.1",
+        "friendsoftypo3/phpstan-typo3": "^0.9.0",
+        "georgringer/numbered-pagination": "^2.1",
+        "phpstan/phpdoc-parser": "^1.29.0",
+        "phpstan/phpstan": "^1.12.21",
+        "phpstan/phpstan-phpunit": "^1.4.0",
+        "phpunit/phpunit": "^10.5.45",
+        "sbuerk/fixture-packages": ">=0.1.1 < 2.0.0",
+        "typo3/tailor": "^1.6",
+        "typo3/testing-framework": "^8.2.7"
+    },
+    "repositories": {
+        "packages-dev": {
+            "type": "path",
+            "url": "../packages-dev/*",
+            "options": {
+                "versions": {
+                    "fgtclb/academics-monorepo-shared": "2.0.2-dev"
+                }
+            }
+        },
+        "packages": {
+            "type": "path",
+            "url": "../packages/*/*",
+            "options": {
+                "versions": {
+                    "fgtclb/academic-bite-jobs": "2.0.2-dev",
+                    "fgtclb/academic-contacts4pages": "2.0.2-dev",
+                    "fgtclb/academic-jobs": "2.0.2-dev",
+                    "fgtclb/academic-partners": "2.0.2-dev",
+                    "fgtclb/academic-persons": "2.0.2-dev",
+                    "fgtclb/academic-persons-edit": "2.0.2-dev",
+                    "fgtclb/academic-persons-sync": "2.0.2-dev",
+                    "fgtclb/academic-programs": "2.0.2-dev",
+                    "fgtclb/academic-projects": "2.0.2-dev",
+                    "fgtclb/category-types": "2.0.2-dev",
+                    "fgtclb/academic-base": "2.0.2-dev"
```

Comment:
Same as above.

Reply:
sorting is unrelevant here. Currently not having jq with resorting at hand and "doing it manually" is forbidden.

Maybe at some point when having a "tool" for that in place.